<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="Source">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>491</width>
      <height>641</height>
     </rect>
    </property>
    <property name="title">
     <string>Source</string>
    </property>
    <widget class="QGroupBox" name="Source_PowerSuppliesControl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>451</width>
       <height>471</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Power Supplies Control</string>
     </property>
     <widget class="QPushButton" name="Source_SendValues">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>420</y>
        <width>341</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Send Values</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="Energy">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>391</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Energy</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>351</width>
         <height>62</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="grid_Energy">
        <item row="0" column="1">
         <widget class="QLabel" name="label_Energy_ActualVoltage">
          <property name="text">
           <string>Actual Voltage (V)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSpinBox" name="Energy_TargetVoltage">
          <property name="maximum">
           <number>10000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_Energy_EmissionCurrent">
          <property name="text">
           <string>Emission Current (µA)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="Energy_EmissionCurrent">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="Energy_ActualVoltage">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_Energy_TargetVoltage">
          <property name="text">
           <string>Target Voltage (V)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="Energy_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>391</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Extractor</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>351</width>
         <height>62</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="grid_Extractor">
        <item row="0" column="1">
         <widget class="QLabel" name="label_Extractor_ActualVoltage">
          <property name="text">
           <string>Actual Voltage (V)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSpinBox" name="Extractor_TargetVoltage">
          <property name="minimum">
           <number>-10000</number>
          </property>
          <property name="maximum">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_Extractor_EmissionCurrent">
          <property name="text">
           <string>Leakage Current (µA)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="Extractor_EmissionCurrent">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_Extractor_TargetVoltage">
          <property name="text">
           <string>Target Voltage (V)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="Extractor_ActualVoltage">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="Energy_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>391</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Suppressor</string>
      </property>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>351</width>
         <height>62</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="grid_Suppressor">
        <item row="0" column="1">
         <widget class="QLabel" name="label_Supressor_ActualVoltage">
          <property name="text">
           <string>Actual Voltage (V)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSpinBox" name="Suppressor_TargetVoltage">
          <property name="minimum">
           <number>-10000</number>
          </property>
          <property name="maximum">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_Suppressor_EmissionCurrent">
          <property name="text">
           <string>Leakage Current (µA)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="Suppressor_EmissionCurrent">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="Suppressor_ActualVoltage">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_Supressor_TargetVoltage">
          <property name="text">
           <string>Target Voltage (V)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="Source_Connection_Init">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>451</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Connection Init</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>351</width>
        <height>54</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QComboBox" name="Source_COM_Choice">
         <item>
          <property name="text">
           <string>COM1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM7</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="Source_Connect">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="Source_COM_Status">
         <property name="text">
          <string>Status</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="Source_Gun">
         <property name="text">
          <string>Gun ON</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="Source_Gun_Status">
         <property name="text">
          <string>Status</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>491</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string>Lenses</string>
    </property>
    <widget class="QGroupBox" name="L2_Connection_Init">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>451</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Connection Init</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>351</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="L2_COM_Choice">
         <item>
          <property name="text">
           <string>COM1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM7</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="L2_Connect">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="L2_COM_Status">
         <property name="text">
          <string>Status</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="Lenses_PowerSuppliesControl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>451</width>
       <height>341</height>
      </rect>
     </property>
     <property name="title">
      <string>Power Supplies Control</string>
     </property>
     <widget class="QGroupBox" name="L1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>391</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>L1</string>
      </property>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>351</width>
         <height>62</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="grid_L1">
        <item row="0" column="1">
         <widget class="QLabel" name="label_L1_ActualVoltage">
          <property name="text">
           <string>Actual Voltage (V)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSpinBox" name="L1_TargetVoltage">
          <property name="maximum">
           <number>10000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_L1_LeakageCurrent">
          <property name="text">
           <string>Leakage Current (µA)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="L1_LeakageCurrent">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="L1_ActualVoltage">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_L1_TargetVoltage">
          <property name="text">
           <string>Target Voltage (V)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="L2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>391</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>L2</string>
      </property>
      <widget class="QWidget" name="layoutWidget_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>351</width>
         <height>62</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="grid_L2">
        <item row="0" column="1">
         <widget class="QLabel" name="label_L2_ActualVoltage">
          <property name="text">
           <string>Actual Voltage (V)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSpinBox" name="L2_TargetVoltage">
          <property name="maximum">
           <number>10000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_L2_LeakageCurrent">
          <property name="text">
           <string>Leakage Current (µA)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="L2_LeakageCurrent">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="L2_ActualVoltage">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_L2_TargetVoltage">
          <property name="text">
           <string>Target Voltage (V)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="Lenses_SendValues">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>300</y>
        <width>351</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Send Values</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="Quit">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>570</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1042</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
